Core/src/org/sleuthkit/autopsy/logicalimager/dsp/LogicalImagerDSProcessor.java

```diff
@@ -170,9 +171,21 @@ public void run(DataSourceProcessorProgressMonitor progressMonitor, DataSourcePr
         }
         File src = imageDirPath.toFile();
 
+        try {
+            progressMonitor.setProgressText(Bundle.AddLogicalImageTask_copyingImageFromTo(src.toString(), dest.toString()));
+            FileUtils.copyDirectory(src, dest);
```

Comment:
This method is called on the EDT and will freeze the UI while copying the directory. The copy needs to happen in the task which is on a different thread.